app/pages/lab/subject-sets.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Link } from 'react-router';
+import LoadingIndicator from '../../components/loading-indicator';
+import Paginator from '../../talk/lib/paginator';
+
+const SubjectSetsPage = (props) => {
+  const meta = props.subjectSets.length ? props.subjectSets[0].getMeta() : {};
+
+  return (
+    <div>
+      <div className="form-label">Subject sets</div>
+      <small>
+        Subject sets are a group of data presented to volunteers in a project. A subject is typically
+        an image, graph, photo, audio recording, video, or a collection of these different things.
```

Comment:
We don't support all of these types yet in the subject viewer in the classifier nor allow all these types to be uploaded in the lab UI. I think it's best to remove that second sentence.